### PR TITLE
Merge assets package

### PR DESCRIPTION
**Pull request by nbey**

### CODE REVIEW

File: sencha-workspace/packages/local/slate-assets/src/assets/controller/Assets.js [hunk 1]
```diff
@@ -0,0 +1,1352 @@
...
+    showAsset: function(assetId) {
+        var me = this,
+            ExtHistory = Ext.util.History,
+            manager = me.getManager(),
+            id = ExtHistory.decodeRouteComponent(assetId),
+
+            _onAssetNotFound = function() {
+                ExtHistory.resumeState(false);
+//                Ext.resumeLayouts(true);
+                debugger;
```
Comment (themightychris) [inline sencha-workspace/packages/local/slate-assets/src/assets/controller/Assets.js]: @nbey do you recall why this `debugger` got left in here? Was there something you were still working on here we should leave a note about before removing this?

Comment (nbey) [inline sencha-workspace/packages/local/slate-assets/src/assets/controller/Assets.js]: @themightychris I can't recall -- it's likely related to suspending/resuming the layouts, but I think it should be fine to remove.

File: sencha-workspace/packages/slate-assets/src/assets/controller/Assets.js [hunk 2]
```diff
@@ -376,6 +376,7 @@ Ext.define('Slate.assets.controller.Assets', {
         Ext.Ajax.request({
```
Comment (themightychris) [inline sencha-workspace/packages/slate-assets/src/assets/controller/Assets.js]: @KClough this should just be:

``` javascript
Slate.API.request({
    url: '/assets/'+asset.getId()+'/activity',
    params: {
        'include[]': [
            'Actor',
            'Media',
            'changes'
        ]
    },
    success: function(response, opts) {
        var responseData = Ext.decode(response.responseText);
        asset.beginEdit();
        asset.set('Stories', responseData.data);
        asset.endEdit();
        asset.commit();
        return manager.updateSelectedAsset(asset);
    }
});
```

`Ext.Ajax` should never be used directly

Comment (themightychris) [inline sencha-workspace/packages/slate-assets/src/assets/controller/Assets.js]: @KClough also all those `asset.` calls can be done at once: `asset.set('Stories', responseData.data, { commit: true })`

**Pull request merged**